src/containers/Chat/ChatMessages/ChatMessages.tsx

```diff
@@ -344,6 +359,15 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({ contactId }) => {
     updateConversationsCache(allConversationsCopy, client, queryVariables);
   };
 
+  // conversationInfo should not be empty
+  if (!Object.prototype.hasOwnProperty.call(conversationInfo, 'contact')) {
```

Comment:
I think you can also do this
```suggestion
  if (!conversationInfo.hasOwnProperty('contact')) {
```

Reply:
It has been updated by the eslint rules, please review - https://eslint.org/docs/rules/no-prototype-builtins

Reply:
Makes sense. Looking good.